### DISCUSSION TITLE
RNASeq: the grouchy grinch Ch 4: Counting troubles

### DISCUSSION THREAD

**Discussion by nirjharaloy** [biostars/biostar-handbook-issues#189] [General]
Hi, I was not able to run this code. I activated bioinfo. Please help.
$ featureCounts -t exon -g Parent -a refs/grinch-annotations_2.gff -o counts.txt bam/Cranky1.bam

**ERROR: invalid parameter: 'bam/Cranky1.bam'**

**Comment by njbowen** [comment]
I ran through the section and copied and pasted your command exactly and it worked.  
make sure you are in the directory that contains the bam directory.   that is, enter "ls" from your bash prompt and make sure you can see the bam directory that contains your alignments from the previous step.

**Comment by nirjharaloy** [comment]
Thanks. It has the bam directory. Still it isn't working.  
$ ls
bam  grinch.tar.gz  ids  reads  refs
(bioinfo)

**Comment by ialbert** [comment]
I have now rerun the example and it works.  Make sure to check that the actual file bam/Cranky1.bam is there. 

Your `ls` commands does not show that. If I type an invalid file name I get a similar error as you:

```
featureCounts -a refs/grinch-annotations_4.gtf -o counts.txt bam/Cranky1000.bam

ERROR: invalid parameter: 'bam/Cranky1000.bam'
```

I have also noticed a similar error reported here:

https://support.bioconductor.org/p/9138099/

where it says that if you copy-paste from PDF you may end up with an incorrect character and that leads to a similar error.

**Comment by nirjharaloy** [comment]
Hello Dr. Albert,

I copied it from the online version, not the PDF. Cranky1.bam file is in the bam directory. my working directory is like this:
work/grinch.

I'm getting this:
$ featureCounts -t exon -g Parent -a refs/grinch-annotations_2.gff -o counts.txt bam/Cranky1.bam

ERROR: invalid parameter: 'bam/Cranky1.bam'

**Reply by molecules** [reply]
If you type in that command (instead of copy/pasting it), do you get the same error message?
